### ISSUE TITLE
mask after normalization

### ISSUE THREAD

**Issue by Akihito233** [ZhengYinan-AIR/Diffusion-Planner#36]
Hello author, I have a question about normalization. Values ​​with non-zero mean will not be 0 after normalization. For example, if the x mean of the agent or lane is 10, then 0 will be -0.5 after normalization. So, is there a problem with the multiple calculations of the mask operation in the model encoder and decoder? Will the non-existent part that should be masked avoid masking because the value is no longer 0? Refer to the code "neighbor_current_mask = torch.sum(torch.ne(neighbors_current[..., :4], 0), dim=-1) == 0"
